docs/dev/internals/issue-workflow.md

```diff
@@ -26,50 +26,62 @@ description: "How Contao developers are supposed to handle issues and PRs on Git
     }
 </style>
 
-## Handling new issues
+This article describes the issue workflow used on the [official monorepository of Contao on GitHub](https://github.com/contao/contao).
 
-After reading a new issue, please label it either as <span class="label-bug">bug</span> or as
+## Handling of new issues
+
+After you have submitted a new issue, one of the maintainers will label it either as <span class="label-bug">bug</span> or as
 <span class="label-feature">feature</span>.
 
-If it is a bug, please also assign the ticket to the milestone of the version in which the bug needs to be fixed (e.g.
-`4.4`). Do not assign feature requests to a milestone. 
+If it is a bug, they will assign your issue to the milestone of the minor version in which the bug needs to be fixed (e.g.
+`4.4`). Maintainers will not assign feature requests to any milestone.
 
 New issues do not get an assignee by default.
 
-## Handling new PRs
+## Handling of new Pull Requests
 
-After reading a new PR, please label it either as <span class="label-bug">bug</span> or as
+After you have submitted a new Pull Request, one of the maintainers will label it either as <span class="label-bug">bug</span> or as
 <span class="label-feature">feature</span>.
 
-If the PR is targeted against an active version branch (e.g. `4.4`), please assign it to the corresponding milestone.
-Do not assign PRs targeted against the `master` branch to a milestone.
+If the Pull Request targets a currently actively supported version branch (e.g. `4.4`) (bugs only), the maintainer will assign
+it to the corresponding milestone.
+For Pull Requests that target the `master` branch (usually features), the maintainers will not assign any milestone.
 
-Please assign new PRs to their creator.
+If you are the creator of a Pull Request, the maintainer will assign it to you.
 
 ## Reviewing bug reports
 
-If you cannot reproduce the bug, please add the <span class="label-status">unconfirmed</span> label to the ticket. If
-it is unclear how to fix the bug, please add the <span class="label-discuss">up for discussion</span> label.
+If a maintainer cannot reproduce the bug, they will add the <span class="label-status">unconfirmed</span> label to the issue.
+If it is unclear how to fix the bug, they will add the <span class="label-discuss">up for discussion</span> label which
+means they will discuss possible solutions during periodical phone calls on Mumble which is often a lot more efficient.
```

Comment:
> … label which means … which is often …

This sentence is longish, don't you think? I would maybe remove the "which is often a lot more efficient" part.